README.md

```diff
@@ -3,39 +3,43 @@
 [![Quality](https://img.shields.io/badge/quality-production-green)](https://curity.io/resources/code-examples/status/)
 [![Availability](https://img.shields.io/badge/availability-binary-blue)](https://curity.io/resources/code-examples/status/)
 
-Curity Identity Server publishes prometheus-compliant metrics. Grafana is a tool that can be used to visualize the metrics collected by Prometheus.
-This repository provides a preconfigured dashboard that shows the most important metrics of Curity Identity Server. Feel free to download the .json file and import it into your Grafana instance to view all the stats of your Curity Identity Server instance.
+Curity Identity Server publishes Prometheus-compliant metrics. Grafana is a tool that can be used to visualize the metrics collected by Prometheus.
+This repository provides a preconfigured dashboard that shows the most important metrics of the Curity Identity Server. Feel free to download the `idsvr-dashboard.json` file and import it into your Grafana instance to view all the stats of your Curity Identity Server instance.
 
 # Prerequisites
 
 To use this dashboard, you must have an environment that has at least the following systems installed:
 
 * An instance of Curity Identity Server with metrics enabled (enabled by default and listening on port 4466)
-* An instance of Prometheus that polls the metrics from Curity Identity Server.
+* An instance of Prometheus that polls the metrics from the Curity Identity Server.
 Check out [Getting Started with Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/)
-* An instance of Grafana with a datasource configured that points to the Prometheus instance above:
+* An instance of Grafana with a data source configured that points to the Prometheus instance above:
 Check out [Getting Started with Grafana](https://grafana.com/docs/grafana/latest/guides/getting_started/)
 
 # The Curity Identity Server Version
 
 Since version 2.0. this dashboard is configured to work with versions of the Curity Identity Server from 7.0. onwards.
 For previous versions of the Curity Identity Server use version 1.0.2 of the dashboard.
 
+Version 3.0 of this dashboard was updated to include metrics introduces in the Curity Identity Server 7.4.0.
```

Comment:
-Which ones and what is the value to the customer? 
-introduces -> introduced
-7.0. onwards -> 7.0 onwards

I think metrics on JDBC timings etc between the Curity Identity Server and data sources will be of quite a bit of interest to customers, so maybe provide a summary of the most useful things a customer can measure, and link to developer docs if they provide further detail.